javascript-stringify.js

```diff
@@ -229,6 +229,11 @@
       return PRIMITIVE_TYPES[typeof value](value, indent, next);
     }
 
+    // Handle Error objects.
+    if (value instanceof Error) {
```

Comment:
By the way, I also thought on this and I think it's good but I'll make one minor change - check the `toString` instead is `[object Error]`.